project/src/Page2.jsx

```diff
@@ -0,0 +1,25 @@
+import './Page2.css'
+
+const Page2 = () => {
```

Comment:
as an extension, see how you can make the experience between this page and your intro page consistent